code/modules/mining/mine_items.dm

```diff
@@ -153,3 +153,101 @@
 	name = "Mining car (not for rails)"
 	icon_crate = "miningcar"
 	icon_state = "miningcar"
+/*****************************Survival Pod********************************/
+
+
+/area/survivalpod
+	name = "\improper Emergency Shelter"
+	icon_state = "away"
+	requires_power = 0
+	has_gravity = 1
+
+/obj/item/weapon/survivalcapsule
+	name = "bluespace shelter capsule"
+	desc = "An emergency shelter stored within a pocket of bluespace."
+	icon_state = "pill3"
+	icon = 'icons/obj/chemical.dmi'
+	w_class = 1.0
+	var/used = FALSE
+
+/obj/item/weapon/survivalcapsule/attack_self()
+	if(used == FALSE)
+		src.loc.visible_message("The [src] begins to shake. Stand back!")
+		used = TRUE
+		sleep(50)
+		playsound(get_turf(src), 'sound/effects/phasein.ogg', 100, 1)
+		PoolOrNew(/obj/effect/effect/smoke, src.loc)
+		load()
+		qdel(src)
+
+/obj/item/weapon/survivalcapsule/proc/load()
+	var/turf/start_turf = get_turf(src.loc)
+	var/turf/cur_turf
+	var/x_size = 5
+	var/y_size = 5
+	var/list/walltypes = list(/turf/simulated/wall)
+	var/floor_type = /turf/simulated/floor/wood
+	var/room
+
+	//Center the room/spawn it
+	start_turf = locate(start_turf.x -2, start_turf.y - 2, start_turf.z)
+
+	room = spawn_room(start_turf, x_size, y_size, walltypes, floor_type, "Emergency Shelter")
+

```

Comment:
now that you removed the start_turf line after this it will break the code below since start_turf is now (because of line 193) no longer the center turf but the bottom left one, and the cur_turf below use start_turf as if it were the center turf.